docs/modules/ROOT/pages/index.adoc

```diff
@@ -404,13 +404,18 @@ signer-payload-map = {
   ? "role": [1* tstr .size (1..max-tstr-length)], ; one or more strings defining the named actor’s relationship(s) to this C2PA asset
   ? "expected_claim": $hash-map, ; hash of expected partial claim
```

Comment:
I see here we use the term 'partial claim' to acknowledge that this valid claim structure differs from the final claim structure which I think is really helpful. I wonder if it worth clarifying in the text within the 'Calculating the expected claim value' section that the valid claim structure described for evaluating the 'expected claim' will be necessarily different from the value calculated by the claim generator due to the zeroing out of fields. 
This doesn't change anything written, I was thinking more of preventing implementers from simply using the same value in the final manifest signature evaluation as it has also been referred to as the claim_structure. Just a thought.

Reply:
I went a step farther and renamed `expected_claim` to `expected_partial_claim`.